acs-integration-tests/src/test/java/com/ge/predix/test/utils/NewUaaTestUtil.java

```diff
@@ -0,0 +1,171 @@
+    }
+
+    public NewUaaTestUtil(final ACSRestTemplateFactory acsRestTemplateFactory, final String uaaUrl) {
+        this.uaaUrl = uaaUrl;
+        this.tokenUrl = this.uaaUrl + "/oauth/token";
+        this.acsRestTemplateFactory = acsRestTemplateFactory;
+        this.adminRestTemplate = this.acsRestTemplateFactory.getOAuth2RestTemplateForUaaAdmin(this.tokenUrl);
+    }
+
+    public void createAcsAdminClient(final List<String> acsZones, final String clientId, final String clientSecret) {
+
+        BaseClientDetails acsAdminClient = new BaseClientDetails();
+        ArrayList<SimpleGrantedAuthority> authorities = new ArrayList<>();
+        authorities.add(new SimpleGrantedAuthority("acs.zones.admin"));
+        authorities.add(new SimpleGrantedAuthority("acs.attributes.read"));
+        authorities.add(new SimpleGrantedAuthority("acs.attributes.write"));
+        authorities.add(new SimpleGrantedAuthority("acs.policies.read"));
+        authorities.add(new SimpleGrantedAuthority("acs.policies.write"));
+        for (String zone : acsZones) {
+            authorities.add(new SimpleGrantedAuthority("predix-acs.zones." + zone + ".admin"));
+            authorities.add(new SimpleGrantedAuthority("predix-acs.zones." + zone + ".user"));
+        }
+
+        acsAdminClient.setAuthorities(authorities);
+        acsAdminClient.setAuthorizedGrantTypes(Collections.singletonList("client_credentials"));
+        acsAdminClient.setClientId(clientId);
+        acsAdminClient.setClientSecret(clientSecret);
+        acsAdminClient.setResourceIds(Collections.singletonList("uaa.none"));
+        createOrUpdateClient(acsAdminClient);
+    }
+
+    public OAuth2RestTemplate createAcsAdminClientAndGetTemplate(final List<String> acsZones) {
+        if (this.acsRestTemplateFactory == null) {
+            throw new IllegalStateException("ACSRestTemplateFactory is null");
+        }
+        String clientId = "admin-" + acsZones.get(0);
+        String clientSecret = "adminSecret-" + acsZones.get(0);
+        createAcsAdminClient(acsZones, clientId, clientSecret);
+        return this.acsRestTemplateFactory.getOAuth2RestTemplateForClient(this.tokenUrl, clientId, clientSecret);
+    }
+
+    public void createReadOnlyPolicyScopeClient(final String clientId, final String clientSecret, final String zone) {
+        this.createClientWithAuthorities(clientId, clientSecret,
+                Arrays.asList(new SimpleGrantedAuthority("predix-acs.zones." + zone + ".user"),
+                        new SimpleGrantedAuthority("acs.policies.read")));
+    }
+
+    public void createNoPolicyScopeClient(final String clientId, final String clientSecret, final String zone) {
+        this.createClientWithAuthorities(clientId, clientSecret,
+                Collections.singletonList(new SimpleGrantedAuthority("predix-acs.zones." + zone + ".user")));
+    }
+
+    public void createReadOnlyConnectorScopeClient(final List<String> acsZones, final String clientId,
```

Comment:
Change parameter to one ACS zone instead of list similar to createReadOnlyPolicyScopeClient and createNoPolicyScopeClient